install-dev/upgrade/sql/1.7.7.0.sql

```diff
@@ -22,11 +22,32 @@ DELETE FROM `PREFIX_configuration` WHERE `name` = 'PS_PRICE_DISPLAY_PRECISION';
 ALTER TABLE `PREFIX_employee` MODIFY COLUMN `optin` tinyint(1) unsigned DEFAULT NULL;
 
 /* Increase column size */
+UPDATE `PREFIX_hook` SET `name` = SUBSTRING(`name`, 1, 191);
```

Comment:
Is it really needed here? It was `varchar(64)` before so it cannot be > 191.

Reply:
The size of the field was updated to 191 in another pull request.